src/oss/schemas/RemoteConnection.yml

```diff
@@ -0,0 +1,25 @@
+type: object
```

Comment:
Are these going to have remote bucket IDs or something like that associated with them?

Reply:
There will be a separate set of API objects for write-replications, linking source bucket + target remote + target bucket + any other configs. The idea being that:
* There might be replication streams for many buckets, and it'd be annoying to enter the connection info repeatedly (especially when updating an auth token)
* In the future we might want to add more kinds of replications, which could reuse the remote-connection metadata